lib/wallaby/helpers/keys.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Wallaby.Helpers.KeyCodes do
+
+  def json(keys) do
```

Comment:
@keathley this is cribbed from hound.. but i wonder if there's a better way to encode unicode characters as json